frontend/server/src/Controllers/Contest.php

```diff
@@ -4788,6 +4798,9 @@ function ($value): ?string {
                     }
             ],
             'admission_mode',
+            'plagiarism_threshold' => $r->ensureOptionalBool(
+                'check_plagiarism'
+            ) ? 90 : 0;
```

Comment:
You already created a variable with this value, so you can use it.
```suggestion
            'plagiarism_threshold' => $checkPlagiarism ? 90 : 0,
```